### DISCUSSION TITLE
Issue with 60% Keyboard in [Tiling Window Manager], Need Help with Keybindings

### DISCUSSION THREAD

**Discussion by Mohamed-abdalazez** [i3/i3#5866] [Q&A] [answered]
Hello, all! 
I recently switched to a 60% keyboard and I'm encountering an issue with keybindings in **Tiling Window Manager**.
Due to the compact layout, I'm forced to use the `fn` key in conjunction with super for arrow key navigation.
However, this is causing unexpected behavior and not working as intended.
Has anyone else faced a similar challenge with a 60% keyboard, Any advice or insights would be greatly appreciated!

**Comment by okraits** [comment]
Please provide more information and describe the exact issue. What did you expect and what is the current behaviour?

You can use `xev` to find out the keysyms and keycodes of the keys you're pressing. You can then use those keysyms and keycodes in your keyboard bindings.

**Reply by Mohamed-abdalazez** [reply]
Okay,
When switching between windows, I press the `Super` key along with the `Fn` key and the letters `A`, `S`, `D`, and `W`. 
This makes the process seem complicated, and sometimes it doesn't work as intended.
So, Is there a way to simplify i3 shortcuts and customize them to be more user-friendly for me?

**Reply by orestisfl** [reply, marked as answer]
Use a different $mod key than `Super`? https://i3wm.org/docs/userguide.html#_using_i3

**Reply by okraits** [reply]
Like  @orestisfl suggested, you can for example map `Capslock` to `Super` and then use that as the modifier key:

`setxkbmap ... -option caps:super`
